EasyCommands/Program.cs

```diff
@@ -249,6 +310,23 @@ class CommandLine {
             }
         }
 
+        public class Thread {
+            String prefix;
+            Command command;
+            protected String name;
+
+            public Thread(Command command, string prefix, string name) {
+                this.command = command;
+                this.prefix = prefix;
+                this.name = name;
+            }
+
+            public Command GetCommand() => command;
+            public void SetCommand(Command c) => command = c;
```

Comment:
nit: Replace with `public Command Command { get; set; }` on the declaration line.

Reply:
fancy! fixed